src/version.txt

```diff
@@ -1 +1 @@
-__version__ = "0.1.6"
+__version__ = "0.1.7-rc.1"
```

Comment:
I feel like we can probably move this repo to a version 1.0.0 at this point, don't you think?

Reply:
I will bump the version to `1.0.0` after I finalize the version upon approval.

Reply:
It has been done.  @dav3r, can you resolve this conversation.

Reply:
Resolved! 👍